src/user/service.ts

```diff
@@ -274,11 +274,15 @@ function isExistingUser(user: User | NewUser): user is User {
 
 }
 
-function getByHref(href: string): string {
+export function getRelHref(href: string): string {
```

Comment:
`Rel` generally in our hypermedia apis stands for `Relation` not `Relative`. How about getPathName?